File: python/raindrops/raindrops.py
def convert(number):
    sound = ""
    if number % 3 == 0 and number % 5 == 0 and number % 7 == 0:
        sound = "PlingPlangPlong"
    elif number % 3 == 0 and number % 5 == 0:
        sound = "PlingPlang"
    elif number % 3 == 0 and number % 7 == 0:
        sound = "PlingPlong"
    elif number % 3 == 0:
        sound = "Pling"
    elif number % 5 == 0 and number % 7 == 0:
        sound = "PlangPlong"
    elif number % 5 == 0:
        sound = "Plang"
    elif number % 7 == 0:
        sound = "Plong"
    else:
        sound = str(number)
    return sound

File: go/raindrops/raindrops.go
package raindrops

import "strconv"

// Convert takes a number and converts it to a string of raindrops
func Convert(input int) string {
	result := ""
	if input%3 == 0 {
		result += "Pling"
	}
	if input%5 == 0 {
		result += "Plang"
	}
	if input%7 == 0 {
		result += "Plong"
	}
	if result == "" {
		result = strconv.Itoa(input)
	}
	return result
}
